obs-websocket-dotnet/Types/OBSScene.cs

```diff
@@ -2,33 +2,41 @@
 using Newtonsoft.Json.Linq;
 using System.Collections.Generic;
 
-namespace OBSWebsocketDotNet.Types
-{
+namespace OBSWebsocketDotNet.Types {
     /// <summary>
     /// Describes a scene in OBS, along with its items
     /// </summary>
-    public class OBSScene
-    {
+    public class OBSScene {
         /// <summary>
         /// OBS Scene name
         /// </summary>
         [JsonProperty(PropertyName = "sceneName")]
         public string Name;
+        
+        /// <summary>
+        /// Scene name as used by GetCurrentProgramScene
+        /// </summary>
+        [JsonProperty(PropertyName = "currentProgramSceneName")]
+        private string ProgramName {
+            set { Name = value; }
+        }
+
 
         /// <summary>
-        /// Scene Index
+        /// Scene name as used by GetCurrentPreviewScene
         /// </summary>
-        [JsonProperty(PropertyName = "sceneIndex")]
-        public int Index;
-        
+        [JsonProperty(PropertyName = "currentPreviewSceneName")]
+        private string PreviewScene {
+            set { Name = value; }
```

Comment:
Syntax should be: 
`public string PreviewScene { get; set; }`

Reply:
See other occurence